<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
    background-color: #E6F3FF;
}


QStatusBar {
    background-color: rgba(255, 255, 255, 0.8);
    color: #000;
}

QMenuBar {
    background-color: rgba(255, 255, 255, 0.8);
    color: #000;
}

QMenuBar::item {
    padding: 5px 10px;
}

QMenuBar::item:selected {
    background-color: rgba(255, 255, 255, 0.8);
}

QMenu {
    background-color: rgba(255, 255, 255, 0.8);
    color: #000;
}

QMenu::item {
    padding: 5px 20px;
}

QMenu::item:selected {
    background-color: rgba(255, 255, 255, 0.8);
}

QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #CCCCCC;
    border-radius: 3px;
    padding: 5px;
    font-family: Arial, sans-serif;
}

QLineEdit:hover {
    background-color: #F0F0F0;
}

QLineEdit:focus {
    background-color: #FFFFCC;
}

QLabel {
    font-family: &quot;Times New Roman&quot;, serif;
    color: #333333;
    font-size: 14px;
    font-weight: bold;
}


#btn_registrar {
    background-color: #007bff;
    color: #fff;
    font-family: &quot;Roboto&quot;, sans-serif;
    font-size: 14px;
    font-weight: bold;
    padding: 8px 16px;
    border: none;
    border-radius: 5px;
}

#btn_registrar:hover {
    background-color: #0056b3;
    cursor: pointer;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman,serif</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#lbl_titulo {
    font-size: 20px;
}
</string>
       </property>
       <property name="text">
        <string>Formulario Cita</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Salario al cabo de 3 Años</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Salario al cabo de 2 Años</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Salario al cabo de 4 Años</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Salario al cabo de 5 Años</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="le_a6"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="le_a5"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Salario al cabo de 1 Año</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_a4"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Salario al cabo de 6 Años</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_a3"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_a2"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_a1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_salario">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #CCCCCC;
    border-radius: 3px;
    padding: 5px;
    font-family: Arial, sans-serif;
}

QLineEdit:hover {
    background-color: #F0F0F0;
}

QLineEdit:focus {
    background-color: #FFFFCC;
}
</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cual es tu Salario?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_estimar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame {
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 10px;
}

#frame QLabel {
    font-size: 14px;
    font-weight: bold;
    margin-bottom: 5px;
}

#frame QLineEdit {
    padding: 5px;
    border: 1px solid #ccc;
    border-radius: 3px;
}

#frame QPushButton {
    padding: 8px 16px;
    background-color: #007bff;
    color: #fff;
    border: none;
    border-radius: 5px;
    font-size: 14px;
}

#frame QPushButton:hover {
    background-color: #0056b3;
    cursor: pointer;
}
</string>
    </property>
    <property name="text">
     <string>Estimar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
